query-coordinator/src/main/scala/com/socrata/querycoordinator/Secondary.scala

```diff
@@ -48,18 +49,13 @@ class Secondary(secondaryProvider: ServiceProviderProvider[AuxiliaryData],
   }
 
   def isInSecondary(name: String, dataset: String, copy: Option[String]): Option[Boolean] = {
-    // TODO we should either create a separate less expensive method for checking if a dataset
-    // is in a secondary, or we should integrate this into schema caching if and when we
-    // build that.
     for {
       instance <- Option(secondaryProvider.provider(name).getInstance())
       base <- Some(reqBuilder(instance))
-      result <- schemaFetcher(base.receiveTimeoutMS(schemaTimeoutMillis), dataset, copy) match {
-        case SchemaFetcher.Successful(newSchema, _, _, _) =>
```

Comment:
When checking for existence -- even in soda1/2, we never _use_ the found schema for anything.